### ISSUE TITLE
overwriting placeholder option?

### ISSUE THREAD

**Issue by no-wa-ke** [claudetech/riot-form#4]
hello!Thank you for providing this awesome library!
I was wondering if you are planning to add overwriting placeholder option. I see there is a *getPlaceholder* function, was not really sure how to pass custom placeholder value to the input.

Thank you!

**Comment by tuvistavie** [comment]
Hi,
Sorry, this is not enough documented.
If you are simply using the inputs without `rf-form`, you can pass it like this:

```
<rf-text-input input-placeholder="My placeholder" />
```

However, this does not work when using `rf-form`, so
here is how we are doing it in our production code:

```javascript
riotForm.configure({
  formatPlaceholder: function (inputName, formName) {
    return i18n.t(`${formName}.form.inputs.${inputName}.placeholder`);
  },
  formatLabel: function (inputName, formName) {
    return i18n.t(`${formName}.form.inputs.${inputName}.label`);
  }
});

```

`formatPlaceholder` takes the input name and form name, and should return the text
displayed as a placeholder. The default behavior is to capitalize the input name.
 
This setup might be a little overkill if you don't internationalize or anything, so I just
updated the library to allow to pass the placeholder directly to the input model.
It is not documented yet, so please see [the tests](https://github.com/claudetech/riot-form/pull/5/files#diff-11f61e1f933980b42c39132dfb4b2ba1R10) for now.
Basically, you can now write something like this:

```javascript
const form = new Form.Builder('hello')
   .addInput({name: 'foo', inputPlaceholder: 'Some placeholder'})
   .build()
```

This is released as version 0.1.9.

Please let me know if this works for you.

**Comment by no-wa-ke** [comment]
awesome! Thank you so much!
